components/lib/crash/src/main/java/mozilla/components/lib/crash/service/SentryService.kt

```diff
@@ -17,8 +17,8 @@ import io.sentry.event.Event
 import io.sentry.event.EventBuilder
 import io.sentry.event.interfaces.ExceptionInterface
 import mozilla.components.Build
-import mozilla.components.lib.crash.Breadcrumb as crashBreadCrumbs
 import mozilla.components.lib.crash.Crash
+import mozilla.components.support.base.crash.Breadcrumb as crashBreadCrumb
```

Comment:
nit: This has been the case before, but I just noticed it here... A class name that starts with a lower letter is confusing to read in code imo. Can we just name it `CrashBreadCrumb`? :) Or since the original class name doesn't use that camel-casing maybe `CrashBreadcrumb`?

Reply:
Agreed, I'll fix it.  Thanks,